media/css/mozorg/home/fundraiser2018.scss

```diff
@@ -0,0 +1,122 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/button';
+
+
+.c-fundraiser {
+    @include clearfix;
+    @include at2x('/media/img/home/fundraiser2018/background-hearts.png', 125px, 200px);
+    background-color: $color-black;
+    background-position: center center;
+    color: $color-white;
+
+    // hide by default if JS is available to avoid flicker
+    // (if visitor previously dismissed)
+    .js & {
+        display: none;
+    }
+}
+
+.c-fundraiser-title {
+    @include text-display-lg;
+    font-weight: normal;
+    margin-bottom: $spacing-lg;
+
+    @media #{$mq-lg} {
+        margin-bottom: 0;
+    }
+}
+
+.c-fundraiser-amount-input {
+    left: 0;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+
+    &:checked + .mzp-c-button {
+        background: $color-gray-70;
+        font-weight: bold;
+    }
+}
+
+.c-fundraiser-recurring {
+    margin-bottom: $spacing-lg;
+
+    label {
+        display: inline-block;
+
+        &:first-child {
+            @include bidi(((margin-right, $spacing-lg, 0),(margin-left, 0, $spacing-lg)));
+        }
+    }
+}
+
+.c-fundraiser-options {
+    label {
+        display: inline-block;
+        margin: 0 $spacing-sm $spacing-md 0;
```

Comment:
The margin values here could do with a `bidi` mixin?

![screen shot 2018-11-22 at 16 07 33](https://user-images.githubusercontent.com/400117/48913842-8848fd00-ee71-11e8-90fb-98222a3bef83.png)